include/circt/Dialect/Calyx/CalyxOps.h

```diff
@@ -74,6 +74,9 @@ LogicalResult verifyCell(Operation *op);
 /// Returns port information about a given component.
 SmallVector<ComponentPortInfo> getComponentPortInfo(Operation *op);
```

Comment:
Could we change the signature of this function to be `getComponentPortInfo(ComponentOp op);`? (see next comment).